pysteps/tests/test_tracking_tdating.py

```diff
@@ -15,6 +15,54 @@
     ("mch", True),
 ]
 
+arg_names_multistep = ("source", "len_timesteps")
+arg_values_multistep = [
+    ("mch", 6),
+]
+
+
+@pytest.mark.parametrize(arg_names_multistep, arg_values_multistep)
+def test_tracking_tdating_dating_multistep(source, len_timesteps):
+    pytest.importorskip("skimage")
+
+    input, metadata = get_precipitation_fields(
```

Comment:
better to avoid redefining built-in method 'input', can your rename with something else, e.g. "input_fields" ?